doc/source/io.rst

```diff
@@ -5,21 +5,18 @@
 .. ipython:: python
    :suppress:
 
-   import os
    import csv
-   from pandas.compat import StringIO, BytesIO
+   import os
+   import matplotlib.pyplot as plt
+   import numpy as np
    import pandas as pd
+   import pandas.util.testing as tm
+   from pandas.compat import StringIO, BytesIO
    ExcelWriter = pd.ExcelWriter
```

Comment:
It's somehow unrelated to your changes, but if you don't mind removing this line, and replacing `ExcelWriter` by `pd.ExcelWriter` wherever is used, that would be great. I think reassigning this just make things difficult for the users to understand.

And if `import pandas.util.testing as tm` is used just in one code block or section, would be nice to move it there, as it may not be obvious for users to know what `tm` is without the import. If it's used in many places leave it here, and we'll see what we can do later on.